python/ray/_private/usage/usage_constants.py

```diff
@@ -6,7 +6,18 @@
 # The name of a json file where usage stats will be written.
 USAGE_STATS_FILE = "usage_stats.json"
 
-USAGE_STATS_HEADS_UP_MESSAGE = (
-    "Usage stats collection will be enabled by default in the next release. "
-    "See https://github.com/ray-project/ray/issues/20857 for more details."
+USAGE_STATS_ENABLED_MESSAGE = (
+    "Usage stats collection is enabled. To disable this, add `--disable-usage-stats` to the command that starts the cluster, or run the following command:"
```

Comment:
Might make sense to clarify that usage stats are enabled / disabled *globally*. Adding the `--disable-usage-stats` flag to the cluster command makes me -- incorrectly -- assume that usage stats collection might be disabled just for that isolated case.

Reply:
Actually `--disable-usage-stats` is just for the current cluster that the command is starting. `ray disable-usage-stats` is global since it just sets a global config.